azure-deployment/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure Deployment script
+PowerShell script for deploying ARM templates to create the Supply Cahin environment in Azure.
+
+## Pre Requisites
+To run the deployment script, please make sure you have the following pre-requisites:
+1.	PowerShell to run the script.
+2.	Node v8.4 or later.
+3.	Azure subscription.
+
+## Getting Started
+Before running the script open the `deploy.conf` file and set the script configuration values.
+Here is a brief description of the configuration values:
+
+### [General]
+* `deploymentType`: This is the deployment type. I could be one of two types:
+    1.	WebApp: Use this option to deploy less secured environment. This option is used generally for development purposes.
+    2.	ASE: Use this option to deploy a secured environment with App Service Environment. Note: this option costs more that the WebApp option, but it’s strongly recommended for production environments.
+* `namePrefix`: The name prefix of all the resources created by this script, except the Network Interfaces and the Storage Accounts of the Ethereum Consortium resources.
+* `resourceGroupName`: The Resource Group name that will include all the created resources by this script.
+* `resourceGroupLocation`: The geographic location of the resource group. Please choose one of the locations listed [here](https://azure.microsoft.com/en-us/regions/)
+* `deploymentName`: The name of the deployment.
+* `gitSmartContractsRepoURL`: The Smart Contracts repository git URL. Default value: `https://raw.githubusercontent.com/CatalystCode/ibera-supply-chain/`
```

Comment:
keep in mind that we have a task to remove all **iBera** occurrences from all the repo names...
This should be changed here and everywhere once we do.

Reply:
yes sure